static/templates/bot_avatar_row.handlebars

```diff
@@ -23,6 +23,10 @@
             <div class="field">{{t "Username" }}</div>
             <div class="value">{{email}}</div>
         </div>
+        <div class="type">
+            <div class="field">{{t "Bot type" }}</div>
+            <div class="value">{{t type }}</div>
```

Comment:
You can use `i18n.t()` to tag strings for translation in javascript, which is what I'd imagined.  I'm going to switch this to do that since it's clearer..